src/components/Progress/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+    StyledProgressBar,
+    StyledBar,
+    StyledFill,
+    StyledProgressText,
+    StyledFillContainer,
+} from './style/ProgressBar.style';
+import { Position, Type } from '../../../../lib';
+
+interface ProgressBarProps {
+    percent: Readonly<number>;
+    positionText?: Readonly<Position>;
+    className?: string;
+}
+
+export const ProgressBar: React.FC<ProgressBarProps> = (props) => {
+    const { percent, positionText, className } = props;
+
+    let fillType: Type = 'error';
+
+    if (percent > 25) {
+        fillType = 'warning';
+    }
+    if (percent > 50) {
+        fillType = 'primary';
+    }
+    if (percent > 75) {
+        fillType = 'success';
+    }
+
+    return (
+        <StyledProgressBar positionText={positionText} className={className}>
+            {positionText && (
```

Comment:
Zobrazení procent bylo v návrhu zakreslené vpravo ne? Pokud ano, pak bych to udělal jako výchozí hodnotu. Jde o to, že pokud si někdo zavolá standardně komponentu `<ProgressBar percent={...} />` pak se mu nebudou zobrazovat procenta, dokud nevyplní i prop atribut `positionText` (tzn. `<ProgressBar percent={...} positionText={'right'} />`.

Připrav to tak, ať se to nemusí vyplňovat. Kdyby sis nevěděla rady, tak se na mě obrať ale dám ti hint. Něco podobného jsme už řešili u komponenty ProgressCircle a prop atributu `circleSize` :)

Reply:
ok